babycare_product_golive_checks/data/product.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+	<!-- TODO: set to noupdate before release -->
```

Comment:
It is not customary to set this kind of data to noupdate.

Reply:
Fixed.